### PR TITLE
Help for breeze commands contain relevant flags.

### PR DESCRIPTION
**Pull request by potiuk**
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: BREEZE.rst [hunk 1]
```diff
@@ -696,85 +699,308 @@ This is the current syntax for  `./breeze <./breeze>`_:
         and setup. This file is automatically sourced when you enter the container. Database
         and webserver ports are forwarded to appropriate database/webserver so that you can
         connect to it from your host environment.
-  ****************************************************************************************************
+
+  Flags:
+
+  Run 'breeze flags' to see all  available applicable flags.
```
Comment (zhongjiajie) [inline BREEZE.rst]:
```suggestion
  Run 'breeze flags' to see all available applicable flags.
```

File: BREEZE.rst [hunk 2]
```diff
@@ -696,85 +699,308 @@ This is the current syntax for  `./breeze <./breeze>`_:
...
+          Python version used for the image. This is always major/minor version.
+          One of:
+
+                 3.6 3.7
+
+  -a, --install-airflow-version <INSTALL_AIRFLOW_VERSION>
+          If specified, installs airflow directly from PIP released version. This happens at
+          image building time in production image and at container entering time for CI image. One of:
+
+                 1.10.9 1.10.8 1.10.7 1.10.6 1.10.5 1.10.4 1.10.3 1.10.2 master v1-10-test
```
Comment (zhongjiajie) [inline BREEZE.rst]:
```suggestion
                 1.10.10 1.10.9 1.10.8 1.10.7 1.10.6 1.10.5 1.10.4 1.10.3 1.10.2 master v1-10-test
```

Comment (zhongjiajie) [reply BREEZE.rst]: I think we should have a var of bash to get the all release support breeze here, we should not hard code it

Comment (zhongjiajie) [reply BREEZE.rst]: I got it, it just a doc, ignore it.

File: breeze [hunk 3]
```diff
@@ -1188,35 +1242,36 @@ Help commands:
...
-    echo "
-Run '${CMDNAME} flags' to see available flags
+function flag_footer() {
+     echo "
+Run '${CMDNAME} flags' to see all  available applicable flags.
 "
 }
 
-flags() {
+function flag_airflow_variants() {
```
Comment (zhongjiajie) [inline breeze:1263]: 👍 for separate function

File: breeze [hunk 4]
```diff
@@ -1383,27 +1442,77 @@ ${FORMATTED_DEFAULT_PROD_EXTRAS}
...
+$(print_star_line)
+ Choose Airflow variant
+$(flag_airflow_variants)
+
+$(print_star_line)
+ Choose backend to run for airflow
+$(flag_backend_variants)
+
+$(print_star_line)
+  Enable production image
```
Comment (zhongjiajie) [inline breeze]:
```suggestion
 Enable production image
```

File: breeze [hunk 5]
```diff
@@ -1383,27 +1442,77 @@ ${FORMATTED_DEFAULT_PROD_EXTRAS}
...
+$(flag_production_image)
+
+$(print_star_line)
+  Additional actions executed while entering breeze
```
Comment (zhongjiajie) [inline breeze]:
```suggestion
 Additional actions executed while entering breeze
```

File: breeze (hunk 4 above)
Comment (zhongjiajie) [reply breeze]: I think we add one more space here

File: breeze [hunk 6]
```diff
@@ -1383,27 +1442,77 @@ ${FORMATTED_DEFAULT_PROD_EXTRAS}
...
+$(print_line)
+
+Summary of all flags supported by Breeze:
+
+$(print_star_line)
+ Choose Airflow variant
+$(flag_airflow_variants)
+
+$(print_star_line)
+ Choose backend to run for airflow
```
Comment (zhongjiajie) [inline breeze]: Should we unify the key word? like change all `airflow -> Airflow` and `kind -> KinD`?

File: BREEZE.rst (hunk 2 above)
Comment (potiuk) [reply BREEZE.rst]: But you are right I will add it to the list in breeze-complete.

File: breeze (hunk 3 above)
Comment (potiuk) [reply breeze:1263]: Yep. Much nicer.

File: breeze (hunk 4 above)
Comment (potiuk) [reply breeze]: Yep

File: breeze (hunk 5 above)
Comment (potiuk) [reply breeze]: Yep

File: breeze (hunk 6 above)
Comment (potiuk) [reply breeze]: Good point!

File: BREEZE.rst (hunk 1 above)
Comment (potiuk) [reply BREEZE.rst]: Yep.

**Comment by potiuk** [comment]
Fixed all comments! Thanks @zhongjiajie !

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Pull request merged**